Mechanisms_evolution/Mechanisms_Evolution.Rmd

```diff
@@ -9,252 +9,222 @@ output:
 knitr::opts_chunk$set(echo = TRUE,fig.width = 5, fig.height = 3)
 ```
 
-## 1. Getting Started
+## Goals for Lab 2
```

Comment:
I would have goals around data rather than software and commands. eg being able to automatically subset data (rather than using dplyr specifically) and being able to compare datasets to know if they are different (rather than t-test)

Reply:
New goals: 
Write your own commands based on descriptive comments and Lab 1
Use R to calculate the mean of data
Plot line graphs using ggplot
Use colors to show different subsets of data
Use R to compare data using a t-test

Reply:
Be able to...
* Create code that you will understand in the future (i.e. by including comments)
* Calculate summary statistics for various subsets of data (using dplyr in R)
* Plot line graphs (using ggplot)
* Plot different subsets of data together (colors in ggplot)
* Compare two datasets (using a t-test)